docs/user/security/securing-communications/elasticsearch-mutual-tls.asciidoc

```diff
@@ -33,7 +33,7 @@ configures {kib} to authenticate with {es} using a
 needs to map the client certificate's distinguished name (DN) to the appropriate
 `kibana_system` role.
 
-NOTE: Using a PKI realm is a gold feature. For a comparison of the Elastic
+NOTE: Using a PKI realm is a subscription feature. For a comparison of the Elastic
 license levels, see https://www.elastic.co/subscriptions[the subscription page].
```

Comment:
Either replace license with "subscription" or simply remove license.

Reply:
Thanks @gchaps. I updated that link in 9a2ece595bc4d39d8b8ba89acb992ba87fb2f03e.